sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 * Removed the `TableClient.create_batch` method along with the `TableBatchOperations` object. The transactional batching is now supported via a simple Python list of tuples.
 * `TableClient.send_batch` has been renamed to `TableClient.submit_transaction`.
 * Removed `BatchTransactionResult` object in favor of returning an iterable of batched entities with returned metadata.
+* Removed Batching context-manager behavior
+* Changed optional `value` and `type` arguments of `EntityProperty` to required.
+* Renamed `EntityProperty.type` to `EntityProperty.edm_type`.
```

Comment:
Maybe add a note that `EntityProperty` is now a tuple? That will allow users to bypass this type completely and just pass in `(1345987163489571634875, 'Edm.Int64')` if they want to.